examples/player/simid_player.html

```diff
@@ -116,7 +116,17 @@
     }
 
     function switchCreative(value) {
-      document.getElementById('creative_url').value = '../creatives/' + value;
+      const creative_url = document.getElementById('creative_url').value = '../creatives/' + value;
+
+      if (creative_url.includes("nonlinear")){
+        document.getElementById("nonlinear_ad").checked = true;
+        document.getElementById("linear_ad").checked = false;
```

Comment:
i had them set up so that only one is selected at a time, so if you check one it should be unchecking the other one

Reply:
N/A after merge